spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -0,0 +1,33 @@
+= Spring Cloud GCP Logging Starter Example
+
+This sample application demonstrates using the
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging[Spring Cloud GCP Logging Starter] in your code.
+
+== Setup & Configuration
+1. Create a Google Cloud Platform Project
+1. https://cloud.google.com/docs/authentication/getting-started#creating_the_service_account[Create a service account]
+with Trace Append permission. Furnish a new JSON key and then set the credentials using the
+`GOOGLE_APPLICATION_CREDENTIALS` environment variable or
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc[using GCP Starter Core properties].
++
+Alternatively, if you have the https://cloud.google.com/sdk/[Google Cloud SDK] installed and
```

Comment:
I think I would offer this as the first option because it's easier to get started with.